src/ansys/mapdl/core/launcher.py

```diff
@@ -1812,7 +1911,7 @@ def launch_mapdl(
         # configure timeout to be 90% of the wait time of the startup
         # time for Ansys.
```

Comment:
If I'm not mistaken, it seems that you changed that into 100%.
```suggestion

```

Reply:
You are right! Yo definitely do a good job as a reviewer! :)